### DISCUSSION TITLE
Meta_Hours_8 - Afterlife

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#2920] [Meta-Hours]
> Why the "You" in an Afterlife wouldn't really be you

![image](https://user-images.githubusercontent.com/5887929/165917689-1406d5ea-68b5-42c7-923e-c42721380baa.png)

### Discord event link 
- https://discord.com/events/840514076538830888/966665549876719636

### Amsterdot 

![IMG_6189](https://user-images.githubusercontent.com/5887929/166648333-1d80da49-7c96-4c68-bc7c-1184f5ff51b6.jpg)

- we had an amazing first-time experience meeting the first time after months of working together around KodaDot remotely, wish every one contributor at KodaDot was at the place! :) 
- discussed various integrations for KodaDot with other teams & para chain teams, most importantly the upcoming one with Basilisk NFT marketplace launch 
- if you are reading this anon, you should definitely hang out on upcoming Dotsama Prague in early June and late June at Polkadot Decoded to know more community rich ecosystem folks.

### KodaDot

- We have on board a few new folks @petersopko @DeluneClaire @thea-exe and Luuu, I let them introduce themselves later, you can spot them hanging around in our Discord, feel free to bump into them :)
- issues are being merged at the usual pace.
- we are having [Burn rate chart](https://github.com/kodadot/nft-gallery/blob/main/BURN_RATE.md) out by @petersopko 
- updated [Leaderboard](https://github.com/kodadot/nft-gallery/blob/main/LEADERBOARD.md)
- working assigned bot in production 
- new [Readme.md](https://github.com/kodadot/nft-gallery/blob/main/README.md) thanks to @thea-exe !
- All Activity indicators you can now find in [Activity.md](https://github.com/kodadot/nft-gallery/blob/main/ACTIVITY.md)

### Reactive NFTs 
@vikiival might drop some bits 👀
- #2934 

### Subsalt
- We will be making a spin-off of the KodaDot engine for Generative Art to demonstrate Reactive NFTs in the wild, where Frank would help us drive the generative art as got the perfect experience there right on spot! 
- I guess once we will have bigger picture to share, we will drop more information at Meta hours, till then I can recommend sometimes check subsalt channel 

### Formalizing Guilds

- I'm going through an extensive study on what is the best way to expand the autonomy of our Guilds, which would help drive stuff autonomously. To some extent, current Stewardship operations by @cryptodamsky are being managed by him, who took sole authorship and helping growing the KodaDot audience, various Ops &c.
- Another proto-DevGuild is being formed by a code review guild where devs made payouts by their humbleness among other developers to drive the development of KodaDot forward
- @kkukelka will have a close co-op as he seems genuinely interested in helping on this part 💪

### Basilisk NFT marketplace

- Incentivized BSX testing. 
- We will be launching NFT marketplace on Basilisk testing soon and we will be inviting folks to report us bugs that occurred etc
- We will be making form to signup for testing and let you know once it's would be out there in public 

### Preparation for Hybrid hackathon

- Hence we have issues would be nice to help us crack on [labeled AMSDOT](https://github.com/kodadot/nft-gallery/issues?q=is%3Aopen+is%3Aissue+label%3Aamsdot) most of them or those unresolved will be going to our upcoming hybrid hackathon and if you take some to solve earlier, it would really help us as well! :)

### Data Science academy with SubSquid
Probably we've spoiled some hints at Twitter spaces last Monday. The goal would be to help other developers learn to write GraphQL queries and understand the bit data science behind it, it would start as a lightweight course to help other developers onboard stuff and what kind of magic we are doing in KodaDot and help others to learn. Will keep you posted on this! :)

### Hiring
We are still hiring and looking for amazing social capital to join our tribe. Despite our established way of giving people bounties we welcome others to join us as part-time or full-time, once you prove you are a great personal fit and vibe check is there.
Best way is to engage on our Discord and not just blindly DM me without any proof of participation :)

#### Notable positions are :
- Frontend Developers (Vue2 + Vue3)
- Rust developer (parachain Dev & Ops & ink! smart contracts)
- Technical Product Manager
- COO
- UI/UX Designer
- Creative Director
- Developer Advocate 
- Technical Writer
- Growth & Ops 

### Handing mic to others

Looking forward @vikiival @cryptodamsky @DeluneClaire @petersopko and others sharing their insights if they have so :)

**Comment by vikiival** [comment]
**Viki's Meta Minutes**
-

### DEV in numbers:
`is:pr is:merged created:>=2022-04-01`

> 87 PRs merged

### Reactive NFTs:

- As a KodaDot we have visited AmsterDOT organized by HydraDX guys
- [presented ReactiveNFTs](https://www.youtube.com/watch?v=3i1nVK-y424)
- Basically three thigs:
1. Conditional NFTs
2. Hooks
3. Entities

### KodaDot x WASM:

We had a talk with another parachain team to implement Kodadot into the smart contracts and the **ink!** smart contract language.

### Basilisk

- Indexer is implemented
- Refactoring & improving code existing code

### UniQuery (Kodapi)

As we are refactoring the gallery, we are building other library - query builder so if you don't know graphql don't worry `nftById` is much simplier

**Comment by DeluneClaire** [comment]
Hello my koda-friends, 
this is my small Claire's meta. I know it looks like many points but its going to be concise and quick! :-)
Its going to be my first talk here so lets see how its going to go 😄

*) introduction
1) reason for KodaDot instagram 
2) what to expect there 
3) improvement of posts 
4) inspiration 
5) cooperation 
6) future socials

**Comment by damskyftw** [comment]
Damsky`s meta-minutes

- Couple words on coordination and hiring
- DotSama Prague
- Frequent reader is live 
- Interesting times ahead, getting familiar with new OS will hopefully bring new interesting concepts
- Articles
- Sneak peek of developer NFTs


- Meta hours call shall continue with free conversation when we finish all entries and questions
